pkg/worker/webhook_handler.go

```diff
@@ -0,0 +1,158 @@
+package worker
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/hatchet-dev/hatchet/internal/signature"
+	"github.com/hatchet-dev/hatchet/pkg/client"
+)
+
+type WebhookHandlerOptions struct {
+	Secret string
+}
+
+type HealthCheckResponse struct {
+	Actions   []string `json:"actions"`
+	Workflows []string `json:"workflows"`
```

Comment:
Shouldn't `Workflows` be a complete workflows object? We'd want to register the correct concurrency settings, timeout settings, etc, right?

Reply:
This just exists to register the actions for the events – the actual workflow is registered separately and thus all settings like concurrency etc should be registered correctly

Reply:
Got it, where is the actual workflow registered from?

Reply:
In the respective SDKs in the healthcheck response right now